draft-ietf-rats-corim.md

```diff
@@ -1764,11 +1776,39 @@ Reference Value does not match.
 
 ##### Comparison for raw-value entries
 
-> I think this comparison method only works if the entry is at key 4 (because
-there needs to be a mask at key 5). Should we have a Reference Value of this
-which stores [expect-raw-value raw-value-mask] in an array?
+The value stored under `measurement-values-map` key 4 is a raw-value entry with
+optional mask. The mask, if present, is stored under `measurement-values-map` key 5.
```

Comment:
I think this is the only `measurement-values-map` entry that isn't independent. Can we put the mask and raw value together in a single compound value so its semantics can be self-contained so a single key?

Reply:
If there is interest in restructuring, I suggest looking more closely at the Intel Profile which places the value and the mask in a record structure:
```
mask-eq = 1
mask-operator = mask-eq
mask-type = bstr
mask-expression = [ mask-operator, value: mask-type, mask: mask-type ]
tagged-mask-expression = #6.60010( mask-expression )
```
It also includes a mask-operator that facilitates verifier behavior. But that is more than base corim is doing. 

However, we should retain backward compatibility with 4 and 5, so they shouldn't be redefined. They should move to `reserved` status.

Otherwise, the text describing how the verifier processes these values seems reasonable. I don't think this is a high priority to "fix".